lib/cocoapods-packager/builder.rb

```diff
@@ -14,41 +16,52 @@ def initialize(source_dir, static_sandbox_root, dynamic_sandbox_root, public_hea
       @exclude_deps = exclude_deps
     end
 
-    def build(platform, library)
-      if library
-        build_static_library(platform)
-      else
-        build_framework(platform)
+    def build(package_type)
+      if package_type == :static_library
```

Comment:
this can use `case package_type` instead of all the ifs, and probably should have an `else` that raises on unexpected package type

Reply:
I've added `:static_framework` to else case because this is what the current behavior is. If none of `--library`, `--embedded`, or `--dynamic` not given, it generates a static framework without embedding resources.